### PR TITLE
Update docs for release v0.1.6

### PR DESCRIPTION
**Pull request by Huanli-Meng**
Update docs for FunctionMesh release v0.1.6:

- Doc update for doc-required PRs (https://github.com/streamnative/function-mesh/pull/242), (https://github.com/streamnative/function-mesh/pull/234), (https://github.com/streamnative/function-mesh/pull/226)
- Release notes
- sidebar.json file

### CODE REVIEW

**Comment by Huanli-Meng** [comment]
@freeznet @nlu90 @tuteng Please help review the docs when you have time. Thanks.

**Comment by nlu90** [comment]
why do we remove the `LogTopic` related content?

**Comment by freeznet** [comment]
@Huanli-Meng @nlu90 we should only remove the `logTopic` from Source CRD and Sink CRD, but not remove from Function CRD.

File: docs/releases/release-note-0-1-6.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Release notes v0.1.6
+category: releases
+id: release-note-0-1-6
+---
+
+Here are some highlights of this release. For a full list of updates available for Release V0.1.6, check out [here](https://github.com/streamnative/function-mesh/releases/tag/v0.1.6)
```
Comment (Jennifer88huang) [inline docs/releases/release-note-0-1-6.md:7]: Do you need to write "v0.1.6": keep consistent with the title.

File: docs/connectors/io-crd-config/sink-crd-config.md [hunk 2]
```diff
@@ -14,13 +14,12 @@ This table lists sink configurations.
 | `name` | The name of a sink connector. |
 | `classname` | The class name of a sink connector. |
 | `tenant` | The tenant of a sink connector. |
-| `Replicas`| The number of Pulsar instances that you want to run this sink connector. |
+| `Replicas`| The number of Pulsar instances that you want to run this sink connector. By default, the `Replicas` is set to `1`. |
```
Comment (freeznet) [inline docs/connectors/io-crd-config/sink-crd-config.md:17]:
```suggestion
| `Replicas`| The number of instances that you want to run this sink connector. By default, the `Replicas` is set to `1`. |
```

File: docs/connectors/io-crd-config/source-crd-config.md [hunk 3]
```diff
@@ -16,7 +16,6 @@ This table lists source configurations.
 | `tenant` | The tenant of a source connector. |
 | `Replicas`| The number of Pulsar instances that you want to run this source connector. |
```
Comment (freeznet) [inline docs/connectors/io-crd-config/source-crd-config.md:17]: | `Replicas`| The number of instances that you want to run this source connector. By default, the `Replicas` is set to `1`. |

File: docs/function-mesh/function-mesh-crd.md [hunk 4]
```diff
@@ -58,7 +58,6 @@ This is a Function Mesh CRD example, which is used to submit the ElasticSearch s
         className: org.apache.pulsar.functions.api.examples.ExclamationFunction
         replicas: 1
         maxReplicas: 1
-        logTopic: persistent://public/default/logging-function-log
```
Comment (freeznet) [inline docs/function-mesh/function-mesh-crd.md:61]: we are not removing `logTopic` from function CRD.

File: docs/releases/release-note-0-1-6.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+Function Mesh Worker service can forward requests to the Kubernetes cluster. After Function Mesh Worker service is started, users can use the [pulsar-admin](https://pulsar.apache.org/docs/en/pulsar-admin/) CLI tool to manage Pulsar Functions and connectors.
+
+In this release, we add Pulsar Functions and connectors (sinks and sources) status API, including adding Function metrics and the `serviceName` in Function Mesh Worker service. Therefore, users can get status of Pulsar Functions and connectors (sinks and sources).
+
+## Support setting the service account name for running Pods
+
+In release v0.1.4, we introduced the Function Mesh Worker service, which ensures users can use the Pulsar CLI tools, such as the pulsar-admin CLI tool, to configure and manage Pulsar Functions and connectors.
+
+In this release, we expose the expose the `serviceAccountName` option in Pulsar Functions' and connectors' CRDs. Therefore, users can run  Pulsar Functions and connectors using the Function Mesh Worker service.
```
Comment (Jennifer88huang) [inline docs/releases/release-note-0-1-6.md:23]:
```suggestion
In this release, we expose the `serviceAccountName` option in CRDs of Pulsar Functions and connectors. Therefore, users can run Pulsar Functions and connectors using the Function Mesh Worker service.
```

File: docs/function-mesh/run-function-mesh.md [hunk 6]
```diff
@@ -64,7 +64,6 @@ To create a Function Mesh, follow these steps.
         className: exclamation_function.ExclamationFunction
         replicas: 1
         maxReplicas: 1
-        logTopic: persistent://public/default/logging-function-log
```
Comment (freeznet) [inline docs/function-mesh/run-function-mesh.md:67]: we are not removing `logTopic` from function CRD.

File: docs/functions/function-crd.md [hunk 7]
```diff
@@ -16,13 +16,12 @@ This table lists Pulsar Function configurations.
 | `classname` | The class name of a Pulsar Function. |
 | `tenant` | The tenant of a Pulsar Function. |
 | `namespace` | The namespace of a Pulsar Function. |
-| `Replicas`| The number of Pulsar instances that you want to run this Pulsar Function. |
+| `Replicas`| The number of Pulsar instances that you want to run this Pulsar Function. By default, the `Replicas` is set to `1`. |
```
Comment (freeznet) [inline docs/functions/function-crd.md:19]:
```suggestion
| `Replicas`| The number of instances that you want to run this Pulsar Function. By default, the `Replicas` is set to `1`. |
```

File: docs/functions/function-debug.md [hunk 8]
```diff
@@ -66,7 +66,6 @@ As shown in the example above, you can get the logger via `context.getLogger()`
       MaxPendingAsyncRequests: 1000
       replicas: 1
       maxReplicas: 5
-      logTopic: persistent://public/default/logging-function-logs
```
Comment (freeznet) [inline docs/functions/function-debug.md:69]: we are not removing `logTopic` from function CRD.

File: docs/functions/run-function/run-go-function.md [hunk 9]
```diff
@@ -143,7 +143,6 @@ After packaging your Pulsar Go Functions, you can submit your Go Functions to a
           MaxPendingAsyncRequests: 1000
           replicas: 1
           maxReplicas: 5
-          logTopic: persistent://public/default/logging-function-logs
```
Comment (freeznet) [inline docs/functions/run-function/run-go-function.md:146]: we are not removing `logTopic` from function CRD.

File: docs/functions/run-function/run-go-function.md [hunk 10]
```diff
@@ -176,7 +175,6 @@ After packaging your Pulsar Go Functions, you can submit your Go Functions to a
         MaxPendingAsyncRequests: 1000
         replicas: 1
         maxReplicas: 5
-        logTopic: persistent://public/default/logging-function-logs
```
Comment (freeznet) [inline docs/functions/run-function/run-go-function.md:179]: we are not removing `logTopic` from function CRD.

File: docs/functions/run-function/run-java-function.md [hunk 11]
```diff
@@ -173,7 +173,6 @@ After packaging your Pulsar Functions, you can submit your Pulsar Functions to a
           MaxPendingAsyncRequests: 1000
           replicas: 1
           maxReplicas: 5
-          logTopic: persistent://public/default/logging-function-logs
```
Comment (freeznet) [inline docs/functions/run-function/run-java-function.md:176]: we are not removing `logTopic` from function CRD.

File: docs/functions/run-function/run-java-function.md [hunk 12]
```diff
@@ -204,7 +203,6 @@ After packaging your Pulsar Functions, you can submit your Pulsar Functions to a
         MaxPendingAsyncRequests: 1000
         replicas: 1
         maxReplicas: 5
-        logTopic: persistent://public/default/logging-function-logs
```
Comment (freeznet) [inline docs/functions/run-function/run-java-function.md:207]: we are not removing `logTopic` from function CRD.

File: docs/functions/run-function/run-python-function.md [hunk 13]
```diff
@@ -136,7 +136,7 @@ After packaging your Pulsar Functions, you can submit your Pulsar Functions to a
           MaxPendingAsyncRequests: 1000
           replicas: 1
           maxReplicas: 5
-          logTopic: persistent://public/default/logging-function-logs
```
Comment (freeznet) [inline docs/functions/run-function/run-python-function.md:139]: we are not removing `logTopic` from function CRD.

File: docs/functions/run-function/run-python-function.md [hunk 14]
```diff
@@ -169,7 +169,6 @@ After packaging your Pulsar Functions, you can submit your Pulsar Functions to a
         MaxPendingAsyncRequests: 1000
         replicas: 1
         maxReplicas: 5
-        logTopic: persistent://public/default/logging-function-logs
```
Comment (freeznet) [inline docs/functions/run-function/run-python-function.md:172]: we are not removing `logTopic` from function CRD.

**Review by freeznet — APPROVED** [review]

File: docs/releases/release-note-0-1-6.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+
+## Remove the `logTopic` option from source and sink CRDs
+
+Currently, Pulsar connectors do not support the `logTopic` option. Therefore, we remove the `logTopic` option from connector CRDs.
+
+## Support Pulsar Functions and connectors (sinks and sources) status API in Function Mesh Worker service
+
+Function Mesh Worker service can forward requests to the Kubernetes cluster. After Function Mesh Worker service is started, users can use the [pulsar-admin](https://pulsar.apache.org/docs/en/pulsar-admin/) CLI tool to manage Pulsar Functions and connectors.
+
+In this release, we add Pulsar Functions and connectors (sinks and sources) status API, including adding Function metrics and the `serviceName` in Function Mesh Worker service. Therefore, users can get status of Pulsar Functions and connectors (sinks and sources).
```
Comment (freeznet) [inline docs/releases/release-note-0-1-6.md:17]:
```suggestion
In this release, we add Pulsar Functions and connectors (sinks and sources) status API, including adding Function metrics in Function Mesh Worker service. Therefore, users can get status of Pulsar Functions and connectors (sinks and sources).

Along with these changes, we also set `serviceName` in StatefulSet in order to communicate with function instance pods.
```

**Review by freeznet — APPROVED** [review]

File: docs/releases/release-note-0-1-6.md [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+
+## Support setting the service account name for running Pods
+
+In release v0.1.4, we introduced the Function Mesh Worker service, which ensures users can use the Pulsar CLI tools, such as the pulsar-admin CLI tool, to configure and manage Pulsar Functions and connectors.
+
+In this release, we expose the expose the `serviceAccountName` option in Pulsar Functions' and connectors' CRDs. Therefore, users can run  Pulsar Functions and connectors using the Function Mesh Worker service.
+
+## Fix the NPE that is generated when the `AutoAck` option is not set
+
+If the `AutoAck` option is not set in the Pulsar Functions and connectors' CRDs, an Null Pointer Exception (NPE) is generated when creating a function, connector (sink or source) using the pulsar-admin CLI tool. To fix this bug, we set the `AutoAck` option to `true` by default. Therefore, users can successfully create Pulsar Functions and connectors.
```
Comment (Jennifer88huang) [inline docs/releases/release-note-0-1-6.md:27]: in the CRDs of ....

**Comment by Huanli-Meng** [comment]
> @Huanli-Meng @nlu90 we should only remove the `logTopic` from Source CRD and Sink CRD, but not remove from Function CRD.

Got it. Thanks. Will update the document.

**Comment by Huanli-Meng** [comment]
@nlu90 @freeznet @Jennifer88huang , thanks for your comments. Docs are updated accordingly. PTAL again. Thanks

File: docs/connectors/io-crd-config/source-crd-config.md [hunk 17]
```diff
@@ -112,5 +111,6 @@ Function Mesh supports customizing the Pod running connectors. This table lists
 | `TerminationGracePeriodSeconds` | It is the amount of time that Kubernetes gives for a Pod before terminating it. |
 | `Volumes` | It is a list of volumes that can be mounted by containers belonging to a Pod. |
 | `ImagePullSecrets` | It is an optional list of references to secrets in the same namespace for pulling any of the images used by a Pod. |
+| `ServiceAccountName` | Specify the name of the service account which is used to run Pulsar Functions or connectors in the Function Mesh Worker service.|
```
Comment (freeznet) [inline docs/connectors/io-crd-config/source-crd-config.md:114]:
```suggestion
| `ServiceAccountName` | Specify the name of the service account which is used to run Pulsar Functions or connectors.|
```

File: docs/functions/function-crd.md [hunk 18]
```diff
@@ -144,5 +144,6 @@ Function Mesh supports customizing the Pod running function instance. This table
 | `TerminationGracePeriodSeconds` | It is the amount of time that Kubernetes gives for a Pod before terminating it. |
 | `Volumes` | It is a list of volumes that can be mounted by containers belonging to a Pod. |
 | `ImagePullSecrets` | It is an optional list of references to secrets in the same namespace for pulling any of the images used by a Pod. |
+| `ServiceAccountName` | Specify the name of the service account which is used to run Pulsar Functions or connectors in the Function Mesh Worker service.|
```
Comment (freeznet) [inline docs/functions/function-crd.md:147]:
```suggestion
| `ServiceAccountName` | Specify the name of the service account which is used to run Pulsar Functions or connectors.|
```

File: docs/releases/release-note-0-1-6.md [hunk 19]
```diff
@@ -0,0 +1,29 @@
...
+
+In this release, we add Pulsar Functions and connectors (sinks and sources) status API, including adding Function metrics in Function Mesh Worker service. Therefore, users can get status of Pulsar Functions and connectors (sinks and sources).
+
+Along with these changes, we also set the `serviceName` option in StatefulSet to communicate with the Function instance Pods.
+
+## Support setting the service account name for running Pods
+
+In release v0.1.4, we introduced the Function Mesh Worker service, which ensures users can use the Pulsar CLI tools, such as the pulsar-admin CLI tool, to configure and manage Pulsar Functions and connectors.
+
+In this release, we expose the `serviceAccountName` option in the CRDs of Pulsar Functions and connectors. Therefore, users can run Pulsar Functions and connectors using the Function Mesh Worker service.
```
Comment (freeznet) [inline docs/releases/release-note-0-1-6.md:25]:
```suggestion
In this release, we expose the `serviceAccountName` option in the CRDs of Pulsar Functions and connectors. Therefore, users can use the specified ServiceAccount for running Pulsar Functions and connectors' Pod.
```

File: docs/releases/release-note-0-1-6.md [hunk 20]
```diff
@@ -0,0 +1,29 @@
...
+
+Function Mesh Worker service can forward requests to the Kubernetes cluster. After Function Mesh Worker service is started, users can use the [pulsar-admin](https://pulsar.apache.org/docs/en/pulsar-admin/) CLI tool to manage Pulsar Functions and connectors.
+
+In this release, we add Pulsar Functions and connectors (sinks and sources) status API, including adding Function metrics in Function Mesh Worker service. Therefore, users can get status of Pulsar Functions and connectors (sinks and sources).
+
+Along with these changes, we also set the `serviceName` option in StatefulSet to communicate with the Function instance Pods.
+
+## Support setting the service account name for running Pods
+
+In release v0.1.4, we introduced the Function Mesh Worker service, which ensures users can use the Pulsar CLI tools, such as the pulsar-admin CLI tool, to configure and manage Pulsar Functions and connectors.
```
Comment (freeznet) [inline docs/releases/release-note-0-1-6.md:23]:
```suggestion
```

**Review by freeznet — APPROVED** [review]
just added some suggestions. currently, the changes with `ServiceAccountName` is not related to mesh worker service.

**Review by nlu90 — APPROVED** [review]

**Review by Jennifer88huang — APPROVED** [review]

**Pull request merged**